### PR TITLE
QR scanner Open cv

### PR DESCRIPTION
**Pull request by ryuk156**
issue: https://github.com/ankitdobhal/Awesome-Python-Scripts/issues/93

About Pull request :
This pr consist of how we can scan Barcode using open cv  and pyzbar  to decode your Barcode 

Demo:

Your Qr:
image

decoded Text:
image

### CODE REVIEW

**Comment by ryuk156** [comment]
@ankitdobhal  review and merge Please

File: Image-Processing/Qr_scanner/Qr_scanner.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import cv2 ##import the module opencv
+import numpy as np ##import the module numpy
+import pyzbar.pyzbar as pyzbar ##import the module pyzbar
```
Comment (disha2sinha) [inline Image-Processing/Qr_scanner/Qr_scanner.py:3]: Insert a new line

**Review by disha2sinha — CHANGES_REQUESTED** [review]

**Comment by ryuk156** [comment]
@disha2sinha this imports are already in new line I have inserted a new line after all the imports are done review, please

**Review by disha2sinha — APPROVED** [review]

**Comment by ryuk156** [comment]
@ankitdobhal  merge this please

**Pull request merged**